scripts/falco-driver-loader

```diff
@@ -618,9 +630,9 @@ if [[ -z "$MAX_RMMOD_WAIT" ]]; then
 	MAX_RMMOD_WAIT=60
 fi
 
-DRIVER_VERSION="@DRIVER_VERSION@"
+DRIVER_VERSION=${DRIVER_VERSION:-"@DRIVER_VERSION@"}
 DRIVER_NAME=${DRIVER_NAME:-"@DRIVER_NAME@"}
-FALCO_VERSION="@FALCO_VERSION@"
+FALCO_VERSION=${FALCO_VERSION:-"@FALCO_VERSION@"}
```

Comment:
As above!

Reply:
`FALCO_VERSION` is actually used only to print the Falco version (that's implicitly tied to the version of this script since `falco-driver-loader` is shipped in a bundle with Falco.

Since it's only used for internal use and does not affect the script execution in any relevant way, I'd not expose this to the users.

Reply:
Just reverted the change. Thanks!